package.json

```diff
@@ -11,7 +11,6 @@
     "refappSelenium": "mvn test -Dcucumber.filter.tags='@selenium and not @initialSetup'",
     "refapp2VitalsAndTriaging": "mvn test -Dcucumber.filter.tags='@selenium and @vitals'",
     "refapp2ClinicalVisit": "mvn test -Dcucumber.filter.tags='@selenium and @clinicalVisit'",
-    "refapp2UserAccount": "mvn test -Dcucumber.filter.tags='@selenium and @userAccount'",
```

Comment:
Unfortunately script section in `package.json` seems not to support comments otherwise the script `refapp2UserAccount` would be just comment out.